src/Register.js

```diff
@@ -74,39 +71,35 @@ class Register extends React.Component {
   render() {
     return (
       <div className="Register">
-          <h1> Join the Community</h1>
-          <form id="registerForm" onSubmit={this.handleSubmit}>
-            <div class = "form-group form in-line">
-              <input 
-                name="email" 
-                type="email" 
-                placeholder="Email Address" 
-              />
-              <br></br>
-              <input
-                name="password"
-                type="password"
-                value={this.state.password}
-                placeholder="Enter password"
-                onChange={this.handleInputChange}
-              />
-              <br></br>
-              <input
-                name="confirmPassword"
-                type="password"
-                value={this.state.confirmPassword}
-                placeholder="Re-enter password"
-                onChange={this.handleInputChange}
-              />
-              <br></br>
+        <img src="ppFriendsLogo.png"></img>
```

Comment:
Remove logo - please refer to wireframes before adding things.

Reply:
won't be removing logos - some group members liked it as it adds more personality to each page :)